frontend/public/components/utils/webhooks.tsx

```diff
@@ -90,7 +90,7 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
       setWebhookSecrets(_.compact(secrets));
       setLoaded(true);
     });
-  }, [isLoaded, canGetSecret]);
+  }, [isLoaded, canGetSecret, secretNames, namespace]);
```

Comment:
Undo this. This change is not related to this PR.

Reply:
Ahh, yarn lint --fix must have added it.